common/data_refinery_common/models/models.py

```diff
@@ -15,6 +18,8 @@
 
 from data_refinery_common.models.organism import Organism
 
+S3 = boto3.client('s3', config=Config(signature_version='s3v4'))
```

Comment:
`signature_version='s3v4'` is voodoo, so you might want to include a comment line about why this is necessary for connecting to certain regions.